frontend/src/index.pug

```diff
@@ -91,6 +91,9 @@ html
     vuetemplate#v_summary
       include summary.pug
 
+    vuetemplate#v_summary_charts
```

Comment:
should we change the `_` here also? try to make it constant through all the repo?

Reply:
this is irrelevant to the kebab-cased name above. the name here is used for the templates ref only. If you want to make it to `-`  also can

Reply:
I am considering the consistency among all the ref names, do you think it will be confusing if the names are not the same across the files?

Reply:
I feel it is ok. Can consider this as we are trying to align the template name with the file naming convention since we have summary_charts.pug and v_summary_charts.js. Actually for consistency I feel we might should align the js file name with the component name. Now the `v_summary.js` file actually is responsible for the component `vSummary`.